blocks/api/test/serializer.js

```diff
@@ -95,6 +96,10 @@ describe( 'block serializer', () => {
 						content: rawContent,
 					};
 				},
+				encodeAttributes( attributes ) {
+					const { align } = attributes;
+					return { align };
+				},
```

Comment:
don't you think the `metadata( 'align' )` is better for consistency? It's just another way of parsing the attribute.